bumpversion.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -eux
+set -euxo pipefail
```

Comment:
> pipefail

TIL, thanks @anguss00

Reply:
Thanks @jkburges! But the credit for finding this one belongs to @lwgordonimos ;-)